pkg/prog/prog.go

```diff
@@ -110,13 +108,15 @@ func Run(fds [3]*os.File, args []string, programs ...Program) int {
 		}
 	}
 
-	if f.Log != "" {
+	if f.Daemon {
+		// See daemon.Spawn(). We expect our stdout file handle is open on a
+		// unique log file for the daemon to write its log messages.
```

Comment:
I find it more readable to put references at the end; it'd also be better to include the package name too. So I'd write this as:

> We expect our stdout file handle is open on a unique log file for the daemon to write its log messages. See daemon.Spawn() in pkg/daemon.

Reply:
Okay. Done. However, I think this is nitpicking.